com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -474,83 +466,6 @@ internal void InitializeVariables()
             }
         }
 
-#if DEVELOPMENT_BUILD || UNITY_EDITOR
```

Comment:
I moved this to its own self contained class, so we wouldn't have static vars messing up our tests

Reply:
There's other static vars in our project, but I didn't think they would influence our tests. It might be worth it to review though